extensions/websockets-next/deployment/src/test/java/io/quarkus/websockets/next/test/security/HttpUpgradePermissionCheckerTest.java

```diff
@@ -138,8 +139,8 @@ String echo(String message) {
     public static class AdminEndpoint {
 
         @OnOpen
-        String open() {
-            return "ready";
+        Uni<String> open() {
```

Comment:
Considering `io.quarkus.websockets.next.test.security.HttpUpgradePermissionCheckerTest.Checker#canDoReadOnEndpoint` doesn't check `open`, the check is happening on the upgrade, this shouldn't have any relation at all. But that's just FYI, I don't think there is anything wrong about the change.

Reply:
Hm, and what's the actual reason for this change?

Reply:
@rsvoboda Hi Rostislav, can you please reply here to Martin before we merge ? I'm assuming the plan is to increase the test coverage of different `open` response types

Reply:
Exactly, to cover different response type. I missed that, thanks for reminder @sberyozkin